extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketListener.java

```diff
@@ -212,25 +209,13 @@ public void onOpen(WebSocket webSocket, okhttp3.Response response) {
      * Translate the received message and pass it on to the related handler. Additionally, updates the client about
      * successful authentications and source connections.
      *
-     * @param body  The {@link ResponseBody} containing the message received.
+     * @param webSocket The websocket on which the message was received.
+     * @param bodyBytes  The {@link ByteString} containing the message received.
      */
     @Override
-    public void onMessage(ResponseBody body) {
+    public void onMessage(WebSocket webSocket, ByteString bodyBytes) {
```

Comment:
In the IDE, all the `on*()` calls seem to have a complaint from the IDE that your `@Override`s aren't annotated the same way as their super-methods.  Most are missing the `@NonNull`.

Probably better to keep these clean though it may cause issues with the tests that try to pass in `null` `WebSocket` instances a lot.  Probably better to just pass in the appropriate `FalseWebSocket`.